### ISSUE TITLE
All images should be tested with a sample hover/definition call

### ISSUE THREAD

**Issue by chrismwendt** [sourcegraph/lsp-adapter#61]
This would give us a sanity check that the images actually work. Maybe we could use lsp-record #37 😄

**Comment by keegancsmith** [comment]
Yup, I meant to add a basic test for all languages. The point of the tool is then to easily just do `lsp-record test haskell` and if it says yet you know it is still working.
